### PR TITLE
Arabic Support

### PR DESCRIPTION
**Pull request by Semsem-programmer**
This fork supports Arabic LanguageToolTextField, with properties:
1- Support Arabic symbols. Showing correctly.
2- Make LanguageToolTextField right to left alignment possible.

### CODE REVIEW

File: lib/presentation/language_tool_text_field.dart [hunk 1]
```diff
@@ -42,6 +42,8 @@ class LanguageToolTextField extends StatefulWidget {
     this.maxLines = 1,
     this.minLines,
     this.expands = false,
+		this.textAlign = TextAlign.right,
+		this.textDirection = TextDirection.rtl,
```
Comment (yurii-prykhodko-solid) [inline lib/presentation/language_tool_text_field.dart:46]: Manually specifying directionality makes sense for apps that are expected to support multiple languages at once -- but even then I'd consider creating scoped [Directionality](https://api.flutter.dev/flutter/widgets/Directionality-class.html) contexts.

Nevertheless, it's still sensible to have these arguments available.

But if we want them to be part of the main package -- I suggest we make them nullable and optional, `null` as default.
```suggestion
		this.textAlign,
		this.textDirection,
```

File: lib/presentation/language_tool_text_field.dart [hunk 2]
```diff
@@ -92,6 +94,8 @@ class _LanguageToolTextFieldState extends State<LanguageToolTextField> {
           padding: const EdgeInsets.all(_padding),
           child: Center(
             child: TextField(
+							 textAlign: widget.textAlign,
+							textDirection: widget.textDirection,
```
Comment (yurii-prykhodko-solid) [inline lib/presentation/language_tool_text_field.dart:98]: I don't see field declarations for `textAlign` and `textDirection` in the Widget.

File: lib/client/language_tool_client.dart [hunk 3]
```diff
@@ -38,7 +38,8 @@ class LanguageToolClient {
     );
 
     final languageToolAnswer = LanguageToolRaw.fromJson(
-      json.decode(result.body) as Map<String, dynamic>,
+      //json.decode(result.body) as Map<String, dynamic>,
+			json.decode(utf8.decode(result.bodyBytes)) as Map<String, dynamic>,
```
Comment (yurii-prykhodko-solid) [inline lib/client/language_tool_client.dart:42]: No commented-out code in VCS, please.
```suggestion
			json.decode(utf8.decode(result.bodyBytes)) as Map<String, dynamic>,
```

**Review by yurii-prykhodko-solid — CHANGES_REQUESTED** [review]
Nice addition! Left a few comments. Please make sure you run `dart format` on the checked-in code.

**Comment by illia-romanenko** [comment]
I would also suggest to format the code as well - now it looks off.

File: lib/presentation/language_tool_text_field.dart (hunk 1 above)
Comment (yurii-prykhodko-solid) [reply lib/presentation/language_tool_text_field.dart:46]: Correction: we won't be able to make textAlign nullable -- instead we should copy the default value from [TextField constructor](https://api.flutter.dev/flutter/material/TextField/TextField.html).

```suggestion
		this.textAlign = TextAlign.start,
		this.textDirection,
```

Comment (Semsem-programmer) [reply lib/presentation/language_tool_text_field.dart:46]: Ok, this is better!

**Review by solid-yuriiprykhodko — APPROVED** [review]
LGTM!

**Pull request merged**